contracts/contracts/ChatOracle.sol

```diff
@@ -92,6 +152,7 @@ contract ChatOracle {
         callbackAddresses[promptId] = msg.sender;
         promptCallbackIds[promptId] = promptCallbackId;
         isPromptProcessed[promptId] = false;
+        promptType[promptId] = "default";
```

Comment:
can't we have one struct to contain all these fields instead of tracking these mappings separately?